service/component-samples/owner/src/main/java/org/fido/iot/sample/OwnerServerApp.java

```diff
@@ -36,44 +42,49 @@ public static void main(String[] args) {
     Tomcat tomcat = new Tomcat();
     tomcat.setPort(TO2_PORT);
     // set the path of tomcat
-    System.setProperty(OwnerAppConstants.SERVER_PATH,
-        Path.of(OwnerConfigLoader.loadConfig(OwnerAppConstants.SERVER_PATH)).toAbsolutePath()
+    System.setProperty(OwnerAppSettings.SERVER_PATH,
+        Path.of(OwnerConfigLoader.loadConfig(OwnerAppSettings.SERVER_PATH)).toAbsolutePath()
             .toString());
 
-    Context ctx = tomcat.addContext("", null);
+    tomcat.setAddDefaultWebXmlToWebapp(false);
+    Context ctx = tomcat.addWebapp("", System.getProperty(OwnerAppSettings.SERVER_PATH));
 
-    ctx.addParameter(OwnerAppConstants.DB_URL,
-        OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_URL));
-    ctx.addParameter(OwnerAppConstants.DB_USER,
-        OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_USER));
-    ctx.addParameter(OwnerAppConstants.DB_PWD,
-        OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_PWD));
+    ctx.addParameter(OwnerAppSettings.DB_URL,
+        OwnerConfigLoader.loadConfig(OwnerAppSettings.DB_URL));
+    ctx.addParameter(OwnerAppSettings.DB_USER,
+        OwnerConfigLoader.loadConfig(OwnerAppSettings.DB_USER));
+    ctx.addParameter(OwnerAppSettings.DB_PWD,
+        OwnerConfigLoader.loadConfig(OwnerAppSettings.DB_PWD));
 
     ctx.addParameter("db.url",
-        OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_URL));
+        OwnerConfigLoader.loadConfig(OwnerAppSettings.DB_URL));
     ctx.addParameter("db.user",
-        OwnerConfigLoader.loadConfig(OwnerAppConstants.DB_USER));
+        OwnerConfigLoader.loadConfig(OwnerAppSettings.DB_USER));
```

Comment:
Redundant property, can be removed.

Reply:
Missed this out previously. Removed now.